File: 卡码网/Python/test06.py
#6. A+B问题VIII
import sys

def main():
	for line in sys.stdin:
		n=int(line)
		for i in range(n):
			nums=list(map(int,sys.stdin.readline().split()))
			print(sum(nums[1:]))
			if i!=n-1:
				print()

if __name__ == '__main__':
	main()

File: 卡码网/C++/test06.cpp
//6. A+B问题VIII
#include <iostream>
using namespace std;

int main(){
	int n;
	while(scanf("%d\n",&n)!=EOF){
		int m;
		for(int i=0;i<n;i++){
			scanf("%d",&m);
			int sum=0;
			for(int j=0;j<m;j++){
				int tmp;
				scanf("%d",&tmp);
				sum+=tmp;
			}
			printf("%d\n",sum);
			if(i!=n-1) printf("\n");
		}
	}
	return 0;
}